### PR TITLE
VACMS-5137 Benefit Hub Landing Page & VC Coupled POC

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #5137

## Testing done
Visual

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### CODE REVIEW

**Comment by kevwalsh** [comment]
out of my lane here, but i think removing the two editorial workflow related blocks in the right sidebar makes sense for this POC. 

From https://pr5202-2bmbsyxy2depdzdnmgjlfbzh49sepb4h.ci.cms.va.gov/escanaba-vet-center 


Escanaba_Vet_Center___Veterans_Affairs

**Comment by kevwalsh** [comment]
FWIW i went ahead and tried that on the tugboat environment with config, but it's still generating an empty `<aside>` 

Escanaba_Vet_Center___Veterans_Affairs

**Comment by ethanteague** [comment]
@mchelen @acrollet @indytechcook @ElijahLynn @kevwalsh 
## Process
Logic / theme treatment is handled primarily by custom field (and a couple/few block) templates in conjunction with reusable component includes. 

## Shortcuts / Things to address _if_ we go coupled
- Table of contents block isn't smart. Will need to create logic that adds unique id's to each h2, and then builds corresponding anchors in TOC.
- Will need to create a series of node bundle templates, instead of adding exclusion / inclusion logic in generic site page.html.twig template.
- Will need to add logic to components to exclude/include headers and descriptions from form fields - for this POC, a few fields have been modified in config.
- Will need to create a variables file for breakpoints, and other theme elements. This POC _primarily_ addresses desktop experience.

## Time investment
Hubs page = **30 hrs**. Much of this time was spent creating reusable components, and trial and error (figuring out what works, what can be most easily extended, etc.).
Vet center = **20 hrs**. Was able to reuse components and lessons learned from Hubs implementation.

## Takeaways
Initial time investment to create a reusable library of elements / components / things will greatly improve delivery times. Two or three developers putting in a week of planning and code (in the beginning) would really help the effort. This said, most content types could probably be themed in 20 hours (using my experience on VC type / knowing that development time will decrease as more reusable patterns are created).
44 content types x 20 hours = 880 hours. Two devs working this in tandem = 440 linear hours. So, 11 weeks without _any_ distractions / meetings, so probably safe to assume 14 linear weeks. 
**Tldr: Two devs fulltime might be able to pull this off in 3.5 months.**

## Test environment links
https://pr5202-2bmbsyxy2depdzdnmgjlfbzh49sepb4h.ci.cms.va.gov
https://pr5202-2bmbsyxy2depdzdnmgjlfbzh49sepb4h.ci.cms.va.gov/health-care
https://pr5202-2bmbsyxy2depdzdnmgjlfbzh49sepb4h.ci.cms.va.gov/escanaba-vet-center

## Video (Hub page)
https://www.loom.com/share/db99563fd4d94620b7ad3120cc9d593f

## Screenshots (Vet Center)
image
image
image
image
image
image
image
